.code-samples.meilisearch.yaml

```
@@ -847,7 +847,7 @@ getting_started_search_md: |-
   ```
   Example:
   ```swift
-    let client = try! MeiliSearch(Config(hostURL: "http://localhost:7700"))
+    let client = try! MeiliSearch(Config("http://localhost:7700")
```

A `)` is missing

---

```suggestion
    let client = try! MeiliSearch("http://localhost:7700")
```

Thanks, config should have been removed as well :)